pennylane/transforms/decompositions/two_qubit_unitary.py

```diff
@@ -49,9 +49,9 @@
 Edag = E.conj().T
 
 # Helpful to have static copies of these since they are needed in a few places.
-CNOT01 = qml.CNOT(wires=[0, 1]).matrix
+CNOT01 = np.array([[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 0, 1], [0, 0, 1, 0]])
 CNOT10 = np.array([[1, 0, 0, 0], [0, 0, 0, 1], [0, 0, 1, 0], [0, 1, 0, 0]])
-SWAP = qml.SWAP(wires=[0, 1]).matrix
+SWAP = np.array([[1, 0, 0, 0], [0, 0, 1, 0], [0, 1, 0, 0], [0, 0, 0, 1]])
```

Comment:
Great find actually, never would have guessed that we start adding to the questions through here !